Program.cs

```diff
@@ -3,9 +3,17 @@
 using System.Windows.Input;
 using System.Windows.Controls;
 using System.Windows.Media.Imaging;
+using static System.Math;
 
 namespace A25;
 
+class Point2D {
+   public readonly int X;
+   public readonly int Y;
+   public Point2D (double x, double y) => (X, Y) = ((int)(x + 0.5), (int)(y + 0.5));
+   public override string ToString () => $"({X},{Y})";
+}

```

Comment:
This can be a readonly struct record instead of a class as it looks clean:
record struct Point2D (int X, int Y) {
    public override string ToString (..);
}